src/shared/Git-Credential-Manager/Program.cs

```diff
@@ -11,9 +12,12 @@ public static class Program
     {
         public static void Main(string[] args)
         {
+            // Set up timer for TRACE2
+            var applicationStartTime = DateTimeOffset.UtcNow.ToUnixTimeSeconds();
```

Comment:
I just made a comment in Trace2.c about moving all of the time-math into it, but I wanted to say that we should be careful converting UTC times to seconds -- we're measuring mili- and micro-second times and here we're lopping of the time to integer seconds.  There is a To...Milliseconds() method that would be better (inside the WriteFOO() events).

That is assuming that there isn't a way to have the WriteFOO() methods subtract two DateTime values directly and get an elapsed TimeSpan expressed in a microsecond value and then format that as a time-in-seconds floating point value in the JSON event.

Reply:
I think I've got an extension method that's doing what we want here. Let me know what you think.